### ISSUE TITLE
15 Tech Terms from the 90s No One Uses Anymore | Best Life

### ISSUE THREAD

**Issue by tg-z** [tg-z/web-clip#942]
Remembering booting up your computer? Or worrying about Y2K? In the age of the technology, things move pretty fast, and our tech terms—more so than almost any other form of jargon—seems exceptionally embarrassing in retrospect. With that in mind, here are 15 tech terms from the 90s-dot-com-bubble hey day that are guaranteed to feel ancient now. So read on, and we'll catch you on the information superhighway. And for more fun ways to look back, here are the [40 things we should all be thankful for in 2017. ](https://bestlifeonline.com/thankful-things-2017/)

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/maxresdefault-51-1500x1000.jpg?resize=1024%2C683&ssl=1)

No modern-day internet experience can compare to the thrill of hearing the static bee-bee-bee-dsshhhh that signaled your ascent into the worldwide web.

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/maxresdefault-3.jpg?resize=1024%2C576&ssl=1)

Reorganizing the data on your computer in order to get it to work faster. When was that last time you did that?

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/xcsyl43tr82z.jpg?resize=1024%2C768&ssl=1)

2000 was the most exciting New Year's Eve ever, as you counted down expecting software to spontaneously combust at midnight due to Y2K- a computer bug that would cause software to think the year 2000 was the year 1900. Luckily, the new century was welcomed largely without incident.

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/zoolander-1.png?resize=1024%2C590&ssl=1)

Before "restart" you used to use "reboot" to describe the arduous process of putting your computer to sleep and waking it up again to get it to work properly. And for more great trivia, here are [30 words guaranteed to make you sound smarter. ](https://bestlifeonline.com/words-to-sound-smarter/)

![](https://i2.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/cc94df205745b80bfb53a3a17958ce0f.jpg?resize=1024%2C576&ssl=1)

Despite the fact that it has the word "punk" in it, a cypherpunk was actually someone who who advocated the widespread use of strong cryptography (writing in code) as a way of protecting your privacy. Julian Assange has been a major voice in the cypherpunk movement since it began in the 80s.

![](https://i2.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/maxresdefault-1.jpg?resize=1024%2C576&ssl=1)

The word that people used for an Internet company before the tech bubble burst and we switched to "startup."

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/maxresdefault-2.jpg?resize=1024%2C578&ssl=1)

Literally, a citizen of the net, back when we thought of it as a whole different, exotic country.

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/3a711d7fb1366b22230a85a50ae7d121.jpg?resize=1024%2C544&ssl=1)

This just means hacking the telephone system, which isn't as exciting as it sounds.

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/v1.jpg?resize=1024%2C576&ssl=1)

Another vaguely sexual sounding word that simply means "to publish online."

![](https://i0.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/3.png?resize=1024%2C572&ssl=1)

Nowadays, computers just go to sleep to save battery, so gone are the days when you got to pick a nice ocean view or rainforest as your special monitor.

![](https://i2.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/zoolander.jpg?resize=1024%2C424&ssl=1)

A palmtop is a "computer small and light enough to be held in your hand." Sound familiar? Back in the '90s the palmtop PC, with its tiny keyboard and adorable little smart pen, was the best way to show people around you that you were a BOSS. Then, the iPhone arrived.

![](https://i1.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/Mission-Impossible-1996-tom-cruise-27898914-1200-785.jpg?resize=1024%2C670&ssl=1)

It's like "gliterrati" (which is a combination of gliterr and literati to denote the intellectual elite) except applied to technology.

![](https://i2.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/Contact.jpg?resize=1024%2C671&ssl=1)

Basically, the Internet, back when we were really, really excited about it.

![](https://i2.wp.com/bestlifeonline.com/wp-content/uploads/2017/11/jurassic_park_15.png?resize=1024%2C576&ssl=1)

The scale for storage is kilo, mega, giga, tera, peta, exa, and zetta. The way we're going, this one might actually catch on one day!

_For more advice on living your best life, _[_follow us on Facebook_](https://www.facebook.com/BestLifeOnline/)_ now!   _ 
 https://bestlifeonline.com/tech-terms-90s-no-one-uses-anymore/
